### PR TITLE
provider/aws: Add ARN to security group data source

### PR DESCRIPTION
**Pull request by grubernaut**
Adds computed `arn` to security group data source

```
$ make testacc TEST=./builtin/providers/aws TESTARGS="-run=TestAccDataSourceAwsSecurityGroup"
==> Checking that code complies with gofmt requirements...
go generate $(go list ./... | grep -v /terraform/vendor/)
2017/05/05 10:17:35 Generated command/internal_plugin_list.go
TF_ACC=1 go test ./builtin/providers/aws -v -run=TestAccDataSourceAwsSecurityGroup -timeout 120m
=== RUN   TestAccDataSourceAwsSecurityGroup
--- PASS: TestAccDataSourceAwsSecurityGroup (56.72s)
PASS
ok      github.com/hashicorp/terraform/builtin/providers/aws    56.725s
```

Fixes: #14237

### CODE REVIEW

**Comment by bflad** [comment]
Thank you!

File: builtin/providers/aws/data_source_aws_security_group.go [hunk 1]
```diff
@@ -81,6 +87,8 @@ func dataSourceAwsSecurityGroupRead(d *schema.ResourceData, meta interface{}) er
 	d.Set("description", sg.Description)
 	d.Set("vpc_id", sg.VpcId)
 	d.Set("tags", tagsToMap(sg.Tags))
+	d.Set("arn", fmt.Sprintf("arn:%s:ec2:%s:%s/security-group/%s",
```
Comment (tbell83) [inline builtin/providers/aws/data_source_aws_security_group.go]: I believe the character before `security-group` should be `:` not `/`

Comment (tbell83) [inline builtin/providers/aws/data_source_aws_security_group.go]: https://github.com/terraform-providers/terraform-provider-aws/pull/910

Comment (tbell83) [inline builtin/providers/aws/data_source_aws_security_group.go]: https://github.com/terraform-providers/terraform-provider-aws/pull/910

**Pull request merged**